src/backend/tests/test-data/projects.test-data.ts

```diff
@@ -152,13 +152,17 @@ export const prismaMaterial: Material = {
 export const prismaManufacturer1: Manufacturer = {
   name: 'Manufacturer1',
   dateCreated: new Date('10-1-2023'),
-  creatorId: 1
+  creatorId: 1,
+  dateDeleted: null,
```

Comment:
These fields are optional so you don't need these nulls

Reply:
I get an error otherwise if they're not present, I'll test further